### ISSUE TITLE
Error using OneAPI v2023.0.0

### ISSUE THREAD

**Issue by yohnk** [intel/intel-extension-for-pytorch#280]
Yesterday I updated a few OneAPI libraries from version 2022.* (there are a couple different versions) to 2023.*. This was just part of my normal system upgrade (Ubuntu 22.04, everything installed with apt).

Today I found that I get the following error when attempting to import the intel_extension_for_pytorch library.

`ImportError: libmkl_sycl.so.2: cannot open shared object file: No such file or directory`

I'm using the setvars.sh script to ensure I've got the correct environment variables.

When searching the OneAPI installation directory for "libmkl_sycl.so" libraries, I find a bunch linked to v3, not v2.

_(base) me@spain:~$ find /opt/intel/ -iname "libmkl_sycl.so.*"
/opt/intel/oneapi/intelpython/python3.9/lib/libmkl_sycl.so.3
/opt/intel/oneapi/intelpython/python3.9/envs/2023.0.0/lib/libmkl_sycl.so.3
/opt/intel/oneapi/mkl/2023.0.0/lib/intel64/libmkl_sycl.so.3_

I've reinstalled PyTorch following the [installation guide](https://intel.github.io/intel-extension-for-pytorch/xpu/1.10.200+gpu/tutorials/installation.html) but that didn't seem to solve the problem.

I'm guessing downgrading is the next step (haven't tried that yet), but wanted to raise this up since the upgrade was just part of normal system maintenance (nightly upgrades) and blocked my workflow today.

Let me know if I can get you anymore information.

**Comment by ZailiWang** [comment]
Hi, would you postpone the update to oneAPI v2023 as current ipex release does not work with v2023 due compatibility issue (as you've seen).
v2023 will work with ipex future releases.

**Comment by yohnk** [comment]
Yep, I sure can. I’d still like to keep this ticket open for two reasons.

First, following the current documentation (linked above) will end up with a broken workflow.

Second,

**Comment by jingxu10** [comment]
Please download the basekit 2022 with the command below at this moment.
`wget https://registrationcenter-download.intel.com/akdlm/irc_nas/18852/l_BaseKit_p_2022.3.0.8767_offline.sh`
Meanwhile, we are working on releasing the next version which works with basekit 2023 in the next week.

**Comment by ShravanBM** [comment]
Thanks for providing link for older version. It helped.

**Comment by jingxu10** [comment]
We have just released a new version of ipex 1.13.10+xpu, works with basekit 2023. Would you please have a try?
https://intel.github.io/intel-extension-for-pytorch/xpu/latest/tutorials/installation.html
Thank you.

**Comment by bouachalazhar** [comment]
I have this : 
`ImportError: libmkl_gnu_thread.so.2: cannot open shared object file: No such file or directory`

System : 
- Asus ZenBook UX325EA Windows 11
- WSL2 Ubuntu 22.04 Jammy
- Python 3.10.6
- IPEX 1.13.10+xpu
- PyTorch 1.13.1
- Intel i7 11th / Intel Iris Xe graphics

**Comment by jingxu10** [comment]
@sanchitintel

**Comment by jingxu10** [comment]
@bouachalazhar  please check my reply in #281

**Comment by DaWe35** [comment]
[I got the same error in this thread](https://github.com/intel/intel-extension-for-pytorch/issues/364#issuecomment-1578518061)
